### PR TITLE
add .github ISSUE_TEMPLATEs

### PR DESCRIPTION
**Pull request by inoas**
You can try and see how it behaves here:

https://github.com/inoas/gleam/issues

### CODE REVIEW

**Comment by inoas** [comment]
I wish there was a way to push ISSUE_TEMPLATE's form values into tags .. didn't find it.

**Comment by inoas** [comment]
I have added a documentation type... if that's a bad idea, let me know and I will remove it.

**Review by lpil — COMMENTED** [review]
Could we remove the labels? I would rather set them myself.

Could we just have bug and other? I would like to keep this as minimal as possible.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+blank_issues_enabled: false
+contact_links:
```
Comment (yordis) [inline .github/ISSUE_TEMPLATE/config.yml:2]: I would suggest pointing people to https://github.com/gleam-lang/gleam/discussions before opening issues related to questions.

File: .github/ISSUE_TEMPLATE/config.yml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  - name: Gleam Community
```
Comment (yordis) [inline .github/ISSUE_TEMPLATE/config.yml:3]:
```suggestion
  - name: Gleam Discord
```

File: .github/ISSUE_TEMPLATE/other.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+name: Other
+description: Create a feature request, enhancement, proposal or suggestion
```
Comment (yordis) [inline .github/ISSUE_TEMPLATE/other.yml:2]:
```suggestion
description: Create a feature request, enhancement, and proposal
```

I would move suggestions or things that are about seeking feedback and validation before creating issues.

**Comment by lpil** [comment]
Is there a way to test this out prior to merging? I'm confused about what the UX will be

**Comment by yordis** [comment]
@lpil https://github.com/inoas/gleam/issues/new/choose

---

P.S: You can add defaults for all the packages at the organization level btw, placing the files under `.github` repo https://github.com/straw-hat-team/.github/tree/master/.github/ISSUE_TEMPLATE)

File: .github/ISSUE_TEMPLATE/config.yml (hunk 1 above)
Comment (inoas) [reply .github/ISSUE_TEMPLATE/config.yml:2]: Hope this is good now as it is (changed a bit)

**Review by lpil — APPROVED** [review]
Wonderful, thank you!!

**Pull request merged**